app/src/main/java/ch/epfl/sdp/replay/IReplayRepository.kt

```diff
@@ -0,0 +1,9 @@
+package ch.epfl.sdp.replay
+
+import ch.epfl.sdp.db.Callback
+import ch.epfl.sdp.game.data.Game
+import java.io.File
+
+interface IReplayRepository {
+    fun getAllGames(cb: Callback<List<ReplayInfo>>)
```

Comment:
I think you should fetch all games for a specific user, not all the games in the database, so getAllGames might need a uid parameter